request-builder.js

```diff
@@ -5,10 +5,17 @@ const _ = require('underscore');
 function buildSlots(slots) {
   const res = {};
   _.each(slots, (value, key) => {
-    res[key] = {
-      name: key,
-      value: value
-    };
+    if( _.isString(value)){
```

Comment:
Stylistic comment: Please add a space between `if()` and `{`. If you setup `eslint` it should warn you